src/stdlib_experimental_io.f90

```diff
@@ -1,28 +1,65 @@
 module stdlib_experimental_io
-use iso_fortran_env, only: sp=>real32, dp=>real64
+use iso_fortran_env, only: sp=>real32, dp=>real64 ,qp=>real128
 implicit none
 private
 public :: loadtxt, savetxt
 
 interface loadtxt
     module procedure sloadtxt
     module procedure dloadtxt
+    module procedure qloadtxt
 end interface
 
 interface savetxt
     module procedure ssavetxt
     module procedure dsavetxt
+    module procedure qsavetxt
 end interface
 
 contains
 
+!PUBLIC
 subroutine sloadtxt(filename, d)
+! Loads a 2D array from a text file.
```

Comment:
For documentation, once we move to use templates as in #35, then the documentation will be written only once, and the templating engine will generate all these `sloadtxt` automatically with the same documentation. Until then, it's fine to copy it by hand.